doc/source/hdf5_storage.rst

```diff
@@ -0,0 +1,60 @@
+HDF5 Storage
+============
+
+Cuds containers can be stored in HDF5 files using the :class:`~.H5File` class. The provided
+api is a currently a reduced version of the Modeling Engine api and supports adding and
+manipulating CUDS containers. Please also note that returned containers from the get
+methods are live proxy objects on top of the HDF5 storage (in contrast to the common
+offline save and read operations).
+
+
+HDF5 Stored Layout
+------------------
+
+Data are stored in HDF5 files using a separate layout for each type of CUDS container.
+The stored layout of the containers is provided bellow using a pseudo-uml description
+for the HDF5 based layout of the data stored in the files.
+
```

Comment:
Do we need a figure or text to illustrate they layout of where the individual containers of Mesh, Lattice and Particles are stored?
(``/particle/foo``, `/lattice/bar`..):